_posts/2017-01-22-Swift-at-Artsy.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: post
+title: "Postmortem: Swift at Artsy"
+date: 2017-01-22 12:18
+author: orta
+categories: [swift, eigen, eidolon, javascript, emission, reactnative]
+series: React Native at Artsy
+---
+
+<center>
+<img src="/images/swift-in-rn/swift-in-react-native.svg" style="width:300px;">
+</center>
+
+Swift became public in June 2014, by August we had started using it in Artsy. By October, we had [Swift in production][eidolon-postmortem] channeling hundreds of thousands of dollars in auction bids. 
+
+It is pretty obvious that Swift is the future of native development on Apple platforms. It was a no-brainer to then build an [appleTV app][emergence] in Swift, integrated Swift-support into our key app Eigen and built non-trivial parts of that [application in Swift][live-a].
+
+We first started experimenting with with React Native in February 2016, and by August 2016, we announced that [Artsy moved to React Native][artsy-rn] effectively meaning new code would be in JavaScript from here onwards.
+
+We're regularly asked _why_ we moved, and it was touched on briefly in our announcement but I'd like to dig in to this and try to cover a lot of our decision process. So, if you're into understanding why a small team of iOS developers with decades of experience switched to JavaScript, read on. 
+
+This post will cover: What are Artsy's apps?, Swifts Positives and Negatives for us, React Native, and Our 1-year Summary.
```

Comment:
The capitalization feels weird here? It's inconsistent ("Our 1-year Summary" as opposed to either "our 1-year summary" or "Our 1-Year Summary"). I'd argue killing capitalization on anything that isn't a proper noun is probably best.